audio/audiofile.go

```diff
@@ -112,7 +112,7 @@ func (f *File) Play() error {
 	// create the audio stream to write to
 	stream, err := portaudio.OpenDefaultStream(0, int(f.AudioData.NumChannels), float64(f.AudioData.SampleRate), BufSize, buf)
 	if err != nil {
-		return err
+		return errors.NewFromErrorCodeInfo(errors.AudioFileNilStream, err.Error());
```

Comment:
This error should be different from the one for `stream.Write`. This one means that portaudio could not open an output stream to write audio data to. The one for `stream.Write` means that portaudio could not write to the output stream. They are distinct errors.